commons/src/main/java/org/dhis2/commons/locationprovider/LocationProviderImpl.kt

```diff
@@ -67,15 +67,10 @@ open class LocationProviderImpl(val context: Context) : LocationProvider {
                 override fun onProviderDisabled(provider: String) {
                     onLocationProviderChanged()
                 }
-
-                @Deprecated("Deprecated in Java")
-                override fun onStatusChanged(provider: String?, status: Int, extras: Bundle?) {
-                    // Need implementation for compatibility
-                }
             }
 
             locationManager.requestLocationUpdates(
-                LocationManager.GPS_PROVIDER,
```

Comment:
We can't do this. NETWORK provider uses wifi access points and cell towers which won't be available in certain locations. I think the best would be combining both, creating two locationRequests, one with the network provider which should be faster and other one with the gps.

Reply:
I have updated the PR including the call to the GPS provider.